### PR TITLE
Strip 'Microsoft.' from various places

### PR DESCRIPTION
**Pull request by cartermp**
fixes https://github.com/fsprojects/fsharp-core-api-docs/issues/4

probably needs some cleaning

### CODE REVIEW

File: generators/apiref.fsx [hunk 1]
```diff
@@ -13,6 +13,12 @@ open FSharp.MetadataFormat
 open Html
 open Apirefloader
 
+let stripMicrosoft (str: string) =
+    if (str.StartsWith("Microsoft")) then
+        str.Split('.').[1..] |> String.concat "."
```
Comment (baronfel) [inline generators/apiref.fsx:18]: trollface suggestion: substring from "Microsoft".length to the end?

Comment (cartermp) [reply generators/apiref.fsx:18]: pft

**Comment by baronfel** [comment]
'microsoft' still appears in the generated urls, we should look into that part too:

image

**Comment by cartermp** [comment]
Fixed generated URLs 
image

**Comment by baronfel** [comment]
Now _this_ is podracing. If you can take care of the conflict I'll give it the ol' mergearoo.

**Comment by cartermp** [comment]
image

On it

**Pull request merged**